core/__tests__/modules/configUser.ts

```diff
@@ -0,0 +1,58 @@
+import fs from "fs";
+import os from "os";
+import { helper } from "@grouparoo/spec-helper";
+
+import { ConfigUser } from "../../src/modules/configUser";
+import { Setting, plugin } from "../../src";
+
+const workerId = process.env.JEST_WORKER_ID;
+const configDir = `${os.tmpdir()}/test/${workerId}/configUser/config`;
+
+process.env.GROUPAROO_CONFIG_DIR = configDir;
+
+describe("modules/ConfigUser", () => {
+  helper.grouparooTestServer({ truncate: true, enableTestPlugin: true });
+
+  beforeEach(async () => {
+    process.env.GROUPAROO_RUN_MODE = "cli:config";
+    const localFile = ConfigUser.localUserFilePath();
+    if (fs.existsSync(localFile)) fs.rmSync(localFile);
+    await Setting.truncate();
+  });
+
+  afterEach(async () => {
+    process.env.GROUPAROO_RUN_MODE = undefined;
+    const localFile = ConfigUser.localUserFilePath();
+    if (fs.existsSync(localFile)) fs.rmSync(localFile);
+  });
+
+  // also tests the get() method
+  test("writes a file to .local/user.json", async () => {
+    expect(fs.existsSync(ConfigUser.localUserFilePath())).toEqual(false);
+    await ConfigUser.create({
+      email: "demo@grouparoo.com",
+      company: "My Company",
+    });
+    expect(fs.existsSync(ConfigUser.localUserFilePath())).toEqual(true);
+    const user = await ConfigUser.get();
+    expect(Object.keys(user)).toEqual(["email"]);
+    expect(user.email).toEqual(true);
+  });
+
+  test("stores the company as the cluster name", async () => {
+    await plugin.registerSetting(
+      "testPlugin",
+      "cluster-name",
+      "My Grouparoo Cluster",
+      "My Grouparoo Cluster",
+      "Name of the cluster",
+      "string"
+    );
```

Comment:
Same note - settings are all created at boot

Reply:
Put this in the code above it, just to be sure:

```js
const count = await Setting.count();
console.log("...", count);
```

And got back `0`. 

Also tried to add the `resetSettings: true` option to the test server in the helper, but that didn't change the result.

Is there something else I have to do to not need this code?

Reply:
That's super weird.   https://github.com/grouparoo/grouparoo/blob/main/core/src/initializers/settings.ts should be creating all the settings in this initializer.  Let's pair on this!